### PR TITLE
[BEAM-1320] Fix warnings in the beam docstrings

### PR DESCRIPTION
**Pull request by sb2nov**
R: @aaltay PTAL

Be sure to do all of the following to help us incorporate your contribution
quickly and easily:

 - [ ] Make sure the PR title is formatted like:
   `[BEAM-<Jira issue #>] Description of pull request`
 - [ ] Make sure tests pass via `mvn clean verify`. (Even better, enable
       Travis-CI on your fork and ensure the whole test matrix passes).
 - [ ] Replace `<Jira issue #>` in the title with the actual Jira issue
       number, if there is one.
 - [ ] If this contribution is large, please file an Apache
       [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.txt).

---

### CODE REVIEW

File: sdks/python/generate_pydoc.sh [hunk 1]
```diff
@@ -61,12 +62,14 @@ cat > target/docs/source/index.rst <<'EOF'
 EOF
 
 # Build the documentation using sphinx
-python $(type -p sphinx-build) -q target/docs/source target/docs/_build -c target/docs/source \
+python $(type -p sphinx-build) -v -a -E -q target/docs/source target/docs/_build -c target/docs/source \
```
Comment (aaltay) [inline sdks/python/generate_pydoc.sh]: Are there long forms for `-v -a -E -q` ?

File: sdks/python/apache_beam/examples/complete/autocomplete.py [hunk 2]
```diff
@@ -78,7 +78,8 @@ def expand(self, words):
             | beam.combiners.Top.LargestPerKey(self._count))
 
 
-def extract_prefixes((word, count)):
+def extract_prefixes(element):
+  (word, count) = element
```
Comment (aaltay) [inline sdks/python/apache_beam/examples/complete/autocomplete.py]: can we remove the parenthesis?

File: sdks/python/generate_pydoc.sh (hunk 1 above)
Comment (sb2nov) [inline sdks/python/generate_pydoc.sh]: Not sure if there are http://www.sphinx-doc.org/en/stable/man/sphinx-build.html

Comment (sb2nov) [inline sdks/python/generate_pydoc.sh]: Added a link to the documentation in the comment

File: sdks/python/apache_beam/examples/complete/autocomplete.py (hunk 2 above)
Comment (sb2nov) [inline sdks/python/apache_beam/examples/complete/autocomplete.py]: Done

**Comment by aaltay** [comment]
LGTM

**Pull request closed without merging**